src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -613,7 +611,9 @@ def get_job_logs(
         try:
             if latest_only:
                 (max_lines, logs) = job.log()
-                if num_lines is not None and max_lines > num_lines:
+                if num_lines is None or max_lines <= num_lines:
+                    first_line = 0
+                else:
```

Comment:
if `num_lines` is unset or greater than the total lines, the first line will be at index 0